composer/loggers/remote_uploader_downloader.py

```diff
@@ -90,6 +90,28 @@ class RemoteUploaderDownloader(LoggerDestination):
             loggers=[remote_uploader_downloader],
         )
 
+    or
+
+    .. testcode:: composer.loggers.remote_uploader_downloader.RemoteUploaderDownloader.__init__
+        from composer.loggers import RemoteUploaderDownloader
+        from composer.utils import LibcloudObjectStore
+        remote_uploader_downloader = RemoteUploaderDownloader(
+            bucket_uri="libcloud://my-bucket",
+            backend_kwargs={
+                'provider': 'google_storage',
+                'container': 'my-bucket',
+                'provider_kwargs=': {
```

Comment:
can you make this the env var example? seems better to default to that i think?

Reply:
done